base_report_to_printer_mail/__openerp__.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```

Comment:
Please use the short header, like in the template : https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py